src/modules/footer/components/attributionInfo/AttributionInfo.js

```diff
@@ -38,24 +39,43 @@ export class AttributionInfo extends BaElement {
 				return html`
             		<div><p>${translate('map_attributionInfo_fallback')}</p></div>
 				`;
-			} 
+			}
 
 			const attributionCopyright = [] ;
+			const attributionsLength = attributions.length;
 
-			attributions.forEach(attribution => {
-				attributionCopyright.push(attribution.copyright.label);
+			attributions.forEach((attribution, index) => {
+				if (attribution.copyright.url  != null) {
+					if (index < attributionsLength - 1) {
+						attributionCopyright.push(html`<a class='attribution-link' target='new' href=${attribution.copyright.url} > ${attribution.copyright.label}, </a>`);
+					}
+					else {
+						attributionCopyright.push(html`<a class='attribution-link' target='new' href=${attribution.copyright.url} > ${attribution.copyright.label} </a>`);
+					} 
+				}
+				else if (attribution.copyright.label != null) {
+					if (index < attributionsLength - 1) {
+						attributionCopyright.push(html`<p class='attribution-label'> ${attribution.copyright.label}, </p>`);
+					}
+					else {
+						attributionCopyright.push(html`<p class='attribution-label'> ${attribution.copyright.label} </p>`);
+					} 
+				} 
```

Comment:
The only difference in each case is the appended semicolon. Can we reduce the redundant code here (-> DRY)?